sub-packages/bionemo-core/src/bionemo/core/data/resources/esm2.yaml

```diff
@@ -40,3 +40,11 @@
   sha256: 006911f92bbc0ded7ea302bbdbfab4c694b409e699c32fd49de1c527a99dba3e  # pragma: allowlist secret
   owner: Peter St John <pstjohn@nvidia.com>
   description: Test data for ESM2 pretraining.
+
+- tag: esm2_inference_testdata:2.0
+  ngc: nvidia/clara/esm2_inference_testdata:2.0  # TODO: upload to NGC
```

Comment:
is this done? will CI pass without it being uploaded?

Reply:
yes, this was already uploaded